gradle/libs.versions.toml

```diff
@@ -1,5 +1,5 @@
 [versions]
-agp = "8.6.1"
+agp = "8.7.2"
 kotlin = "1.9.0"
```

Comment:
I dont want this file to be submitted, because it will not work with my version of Android Studio